_overviews/plugins/index.md

```diff
@@ -42,6 +42,7 @@ include:
 - Alternate compiler back ends such as [Scala.js](https://www.scala-js.org), [Scala Native](http://scala-native.org), and
   [Fortify SCA for Scala](https://developer.lightbend.com/docs/fortify/current/).
 - Linters such as [Wartremover](https://www.wartremover.org) and [Scapegoat](https://github.com/sksamuel/scapegoat).
+<!-- is this even accurate? scalafmt is not a _compiler_ plugin I think, and scalafix relies on the semanticdb plugin? -->
```

Comment:
Removing scalafmt is fine (I had thought it was semanticdb-based, oops!).

Removing scalahost is fine, too.

But it was never my intention to imply, through poor phrasing, that scalafix is a compiler plugin. I wanted to include it because it's so important and because its existence is enabled by semanticdb-scalac, which is a compiler plugin. Not everyone knows what semanticdb is, but everybody knows about scalafix.

Wdyt about the following replacement phrasing:

> Plugins that enable tooling. One such plugin is semanticdb-scalac, which enables [scalafix](https://scalacenter.github.io/scalafix/) (a well-known refactoring and listing tool) to do its work.

Reply:
Semanticdb is currently mentioned next to Paradise, but I guess I can rephrase it to mention scalafix explicitly ;)